src/watchmaker/__init__.py

```diff
@@ -108,30 +135,35 @@ def _get_config_data(self):
                     shutil.copyfileobj(response, outfile)
                 self.config_path = 'config.yaml'
             except urllib.error.URLError:
-                print('The URL used to get the user config.yaml file did not '
-                      'work!\nPlease make sure your config is available.')
+                logging.critical(
+                    'The URL used to get the user config.yaml file did not '
+                    'work!\nPlease make sure your config is available.'
```

Comment:
Thoughts on moving away from multiline messages, now that we've moved to the logging facility? I think they make the log file ugly. If we need multiple lines, we maybe should use multiple `logging()` calls.